parse.go

```diff
@@ -275,7 +276,17 @@ func process(specs []*spec, args []string) error {
 		}
 		if spec.env != "" {
 			if value, found := os.LookupEnv(spec.env); found {
-				err := scalar.ParseValue(spec.dest, value)
+				var err error
+				if spec.multiple {
+					// expect a CSV string in an environment
+					// variable in the case of multiple values
+					values, err := csv.NewReader(strings.NewReader(value)).Read()
+					if err == nil {
+						err = setSlice(spec.dest, values, !spec.separate)
```

Comment:
I think this `err` will ultimately be ignored even if non-nil, because the `values, err := ...` above shadows the outer `err`. Or does it? I'm not sure, but it would definitely be nice to make the code a little clearer. How about just duplicating the `if err != nil` block below three times after the error check on `Read()`, `setSlice()`, and `scalar.ParseValue(...)`